hack/generate-statik.sh

```diff
@@ -38,11 +38,21 @@ elif ! [ -x "$(command -v ${LICENSES})" ]; then
     popd
 fi
 
+# hack for github.com/vbauerster/mpb begin
+if [ -f vendor/github.com/vbauerster/mpb/v5/UNLICENSE ]; then
+   mv vendor/github.com/vbauerster/mpb/v5/UNLICENSE vendor/github.com/vbauerster/mpb/v5/LICENSE
+fi
+
 echo "Collecting licenses"
 cd ${DIR}/..
 ${LICENSES} save github.com/GoogleContainerTools/skaffold/cmd/skaffold --save_path="${TMP_DIR}/skaffold-credits"
 chmod -R u+w "${TMP_DIR}/skaffold-credits"
 
+# hack for github.com/vbauerster/mpb end
```

Comment:
Can you check what is the output of skaffold credits on your branch?
I tried checking the output of skaffold credits and there is no license generation for "github.com/vbauerster" project.
If you don't have LICENSE file, does the tool error out?